pkg/antctl/antctl.go

```diff
@@ -175,6 +175,12 @@ $ antctl get podmulticaststats pod -n namespace`,
   $ antctl get networkpolicy
   Get the list of all control plane NetworkPolicies, sorted by the order in which the policies are evaluated.
   $ antctl get networkpolicy --sort-by=effectivePriority
+  Get the list of all control plane NetworkPolicies, sorted by the Name.
+  $ antctl get networkpolicy --sort-by={.sourceRef.name}
```

Comment:
kubectl doesn't require the braces: https://kubernetes.io/docs/reference/kubectl/cheatsheet/#viewing-and-finding-resources

```
# List PersistentVolumes sorted by capacity
kubectl get pv --sort-by=.spec.capacity.storage
```